src/components/templates/AnimateMap/game/map/entities/createFirebarrelEntity.ts

```diff
@@ -200,7 +200,8 @@ export const createFirebarrelEntity = (
               )
             );
           }
-        }
+        },
+        true
```

Comment:
```suggestion
        }
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2161/commits/470154cca18bb5273e216b7d43acaace4ac25b4a